submissions/Articles/global-payouts-non-profits.md

```diff
@@ -0,0 +1,28 @@
+
+## How Global Payouts Can Make Your Transactions Quick
+
+As cliche as it is to say, “Time is money”, it truly is important for users to get their money on time. Sadly, traditional banking institutions tend to have long delivery processes. For example, [Wise users receives their payment depending on the country and currency](https://wise.com/help/articles/2978029/how-long-does-it-take-to-receive-money-into-a-balance). This can put nonprofits at a disadvantage as it can cause the communities they serve to financially help them in a timely manner.  Unlike their traditional counterparts, global payout systems don’t have a waiting period. Once person sends money, the other person gets its instantly. Nonprofits can benefit greatly from this as they don’t have to worry about the communities they serve not getting the funding they sent.  For instance, [AfricaHacks' collaboration with Chimoney for their hackathon rewards to the winners due to having a feature where they can send people money in just one click](https://chimoney.io/blogs/5-types-of-people-who-should-use-chimoneys-bulk-gift-cards-feature/).  Now before you go, there’s just one more benefit of using global payout systems for nonprofit funding.
+
+## How Global Payouts Can Give You Multiple Ways To Send Money
+
+Global payouts provide the necessary infrastructure for non-profits to send financial support quickly and efficiently. In contrast to traditional banking platforms, global payout platforms users have the freedom to use any method to send their money to other people. This can be helpful to nonprofits as it enables them to decide which delivery method best suits their organization and the people they serve. Case in point, [the International Student Identity Card (ISIC)'s partnership with Chimoney makes it easier for students manage their fiances](https://chimoney.io/blogs/chimoney-partners-with-isic-to-expand-global-benefits-for-chimoney-app-users/) as [they can send and receive money via email, their application, and text messaging](https://chimoney.io/payouts/). Now you know the benefits of global payouts, let’s see how one global payout platform can empower nonprofit and communities.
+
+## How Chimoney Empowers Communities and Non-profits
+
+Before you go to your computer or mobile device and do a long search of global payout platforms, consider checking out Chimoney. In addition to having their own platform where users can facilitate their transactions, they also partner with other platforms to help non-profits reach more clients. For example, [OpenGiving](https://donations.chimoney.io/), a platform helps non-profits and communities around the world can receive donations and get paid easily in various methods applicable to the donor through Chimoney’s payout feature.
```

Comment:
TryOpenGiving is built by Chimoney, so the sentence "they also partner with other platforms to help non-profits reach more clients" may not be entirely accurate. Also, since non-profits primarily receive funding from donors, it would be more appropriate to use "donors" instead of "clients."


Please rewrite this `For example, [OpenGiving](https://donations.chimoney.io/), a platform helps non-profits and communities around the world can receive donations and get paid easily in various methods applicable to the donor through Chimoney’s payout feature.` to `For example, [TryOpenGiving](https://tryopengiving.com/), a platform built by Chimoney helps non-profits and communities around the world to receive donations and get paid easily in various methods applicable to the donor through Chimoney.`